warehouse/query-core/src/main/java/datawave/query/jexl/functions/EvaluationPhaseFilterFunctions.java

```diff
@@ -1342,20 +1289,42 @@ public static boolean excludeRegex(Iterable<?> values, String regex) {
     public static final String TIME_SINCE_EPOCH_FORMAT = "e";
     
     /**
-     * The list of formats attempted At a mimumum the following are found in existing data: "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'", "yyyy-MM-dd HH:mm:ssz",
-     * "yyyy-MM-dd", "yyyy-MM-dd'T'HH'|'mm", "EEE MMM dd HH:mm:ss zzz yyyy"}; "yyyyMMddhhmmss" "yyyyMMddHHmm", "yyyyMMddHH", "yyyyMMdd",
+     * The list of formats attempted At a minimum the following are found in existing data: "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'", "yyyy-MM-dd HH:mm:ssz",
+     * "yyyy-MM-dd", "yyyy-MM-dd'T'HH'|'mm", "EEE MMM dd HH:mm:ss zzz yyyy", "yyyyMMddhhmmss" "yyyyMMddHHmm", "yyyyMMddHH", "yyyyMMdd",
      */
-    protected static final String[] DATE_FORMAT_STRINGS = {"yyyyMMdd:HH:mm:ss:SSSZ", "yyyyMMdd:HH:mm:ss:SSS", "EEE MMM dd HH:mm:ss zzz yyyy",
-            "d MMM yyyy HH:mm:ss 'GMT'", "yyyy-MM-dd HH:mm:ss.SSS Z", "yyyy-MM-dd HH:mm:ss.SSS",
-            "yyyy-MM-dd HH:mm:ss.S Z",
-            "yyyy-MM-dd HH:mm:ss.S",
-            "yyyy-MM-dd HH:mm:ss Z", // ISO 8601
-            "yyyy-MM-dd HH:mm:ssz", "yyyy-MM-dd HH:mm:ss", "yyyyMMdd HHmmss", "yyyy-MM-dd'T'HH'|'mm", "yyyy-MM-dd'T'HH':'mm':'ss'.'SSS'Z'",
-            "yyyy-MM-dd'T'HH':'mm':'ss'Z'", "MM'/'dd'/'yyyy HH':'mm':'ss", "E MMM d HH:mm:ss z yyyy", "E MMM d HH:mm:ss Z yyyy", "yyyyMMdd_HHmmss",
-            "yyyy-MM-dd", "MM/dd/yyyy", "yyyy-MMMM", "yyyy-MMM", "yyyyMMddHHmmss", "yyyyMMddHHmm", "yyyyMMddHH", "yyyyMMdd",};
+    // @formatter:off
+    private static final String[] DATE_FORMAT_STRINGS = {
```

Comment:
This needs to be left protected.  They are accessed in a subclass on the other side.

Reply:
Fixed!